app/models/manager_refresh/inventory_collection/builder/infra_manager.rb

```diff
@@ -18,7 +19,8 @@ def host_networks
 
         def guest_devices
           add_properties(
-            :manager_ref => %i(hardware uid_ems),
+            :manager_ref                  => %i(hardware uid_ems),
+            :parent_inventory_collections => %i(vms),
```

Comment:
Hey sorry should have been more explicit, I think all of these that are currently vms should be vms and miq_templates.

Reply:
done